build/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "node": true
+  }
+}
```

Comment:
_This extends the ESLint configuration defined in the root, only adding the Node.js environment to build-related files. See [Configuration Cascading and Hierarchy](https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy) for more information._